components/network/tor/manifests/sample-manifest.p5m

```diff
@@ -25,13 +25,16 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=etc/tor/torrc.sample
```

Comment:
Should we deliver `etc/tor/torrc.sample`?

Reply:
IMHO it is better to have an example configuration file especially for those users which never have used tor before.
In the sample file many options are explained.

Reply:
Good, let's deliver it then.